spacy/matcher/matcher.pyx

```diff
@@ -51,6 +53,7 @@ cdef class Matcher:
         self.vocab = vocab
         self.mem = Pool()
         self.validate = validate
+        self.fuzzy = fuzzy if fuzzy is not None else 0
```

Comment:
I am not really familiar with the way spacy usually handles normalization, but it is probably worth thinking about whether to use a range `0-100` or `0-1`.